numpy/random/_generator.pyx

```diff
@@ -396,6 +396,41 @@ cdef class Generator:
         out : ndarray or scalar
             Drawn samples from the parameterized beta distribution.
 
+        Examples
+        -------- 
+        If ``a == b`` and both are > 1, the distribution is symmetric with 
+        mean 0.5.
+
+        >>> rng = np.random.default_rng()
+        >>> a, b, size = 2.0, 2.0, 10000
+        >>> sample = rng.beta(a=a, b=b, size=size)
+        >>> np.mean(sample)
+        0.5047328775385895  # may vary
+        
+        Otherwise the distribution is skewed left or right according to
+        whether ``a`` or ``b`` is greater. The distribution is mirror
+        symmetric. See for example:
+        
+        >>> a, b, size = 2, 7, 10000
+        >>> sample_left = rng.beta(a=a, b=b, size=size)
+        >>> sample_right = rng.beta(a=b, b=a, size=size)
+        >>> m_left, m_right = np.mean(sample_left), np.mean(sample_right)
+        >>> m_left, m_right
+        (0.22334081861904062, 0.7749320166133927)  # may vary
+        >>> m_left + m_right  
```

Comment:
Why show this value? I would either omit it, or add with a comment we expect this to be 1.0 (up to statistical fluctuations). Similar for the previous line (where we expect 2/9, 7/9)

Reply:
I made a few changes do you like it more the way it is now?